crossbeam-utils/src/cache_padded.rs

```diff
@@ -9,14 +9,17 @@ use core::ops::{Deref, DerefMut};
 /// CPU cores. Use `CachePadded` to ensure updating one piece of data doesn't invalidate other
 /// cached data.
 ///
-/// Cache lines are assumed to be 64 bytes on all architectures.
-///
 /// # Size and alignment
 ///
-/// The size of `CachePadded<T>` is the smallest multiple of 64 bytes large enough to accommodate
+/// Cache lines are assumed to be of N bytes, depending on the architecture:
+///
+/// * On x86-64, N = 128.
```

Comment:
Isn't it more the case that x64 CPUs fetch two cache lines at a time to align them to 128-bit chunks rather than that a single cache line is 128-bit?

Reply:
Yes. Maybe the wording is not great, but `N` is just a pessimistic *guess* of how long a cache line is. So `N` is at least the length of a cache line, but may be larger than that in order to accommodate architectures with longer cache lines or ones that have unusual prefetching strategies.

Reply:
Okay, perhaps this could be reworded slightly to avoid confusion later? In any case LGTM.

Reply:
Sure - just added an additional comment for clarification.